code/inference.py

```diff
@@ -14,6 +14,8 @@ def model_fn(model_dir):
     else:
         device = -1
 
-    generation = pipeline("text-generation", model=model, tokenizer=tokenizer, device=device)
+    generation = pipeline(
+        "text-generation", model=model, tokenizer=tokenizer, device=device
+    )
 
-    return generation
+    return
```

Comment:
why did you remove `generation` ?  you need to return the model in the `model_fn`

Reply:
Yeah, I must have deleted mistakenly, Updated it now.

Reply:
thanks!